packages/server/lib/controllers/connection.controller.ts

```diff
@@ -558,6 +558,26 @@ class ConnectionController {
                     raw: req.body.raw || req.body
                 };
 
+                if (req.body['oauth_client_id']) {
+                    oAuthCredentials.config_override = {
+                        client_id: req.body['oauth_client_id']
+                    };
+                }
+
+                if (req.body['oauth_client_secret']) {
+                    oAuthCredentials.config_override = {
+                        ...oAuthCredentials.config_override,
+                        client_secret: req.body['oauth_client_secret']
+                    };
+                }
+
+                if (req.body['oauth_scopes']) {
```

Comment:
does those three do not need to be set altogether? 
like what happen if we only set the secret or only the scopes? from what I understand I would expect to be everything or nothing

Reply:
The three do not need to be set altogether, no. Scopes can be edited independently. A user could theoretically just override the client_id and not the secret which is why I separated them out.

Reply:
okay got it, thansk for the clarification 👌🏻